jwst/extract_1d/extract.py

```diff
@@ -2875,14 +2878,16 @@ def do_extract1d(input_model, ref_dict, smoothing_length=None,
                         "so apply_nod_offset will be set to False",
                         source_type)
 
-        if isinstance(input_model, (datamodels.ImageModel,
-                                    datamodels.DrizProductModel)):
+#        if isinstance(input_model, (datamodels.ImageModel,
+#                                    datamodels.DrizProductModel)):
+
+        if isinstance(input_model, (datamodels.ImageModel)):
```

Comment:
Don't need the extra parens around `datamodels.ImageModel`, because the argument now contains only 1 item.